src/clang.zig

```diff
@@ -876,7 +931,16 @@ pub const StringLiteral = opaque {
     extern fn ZigClangStringLiteral_getString_bytes_begin_size(*const StringLiteral, *usize) [*]const u8;
 };
 
-pub const StringRef = opaque {};
+pub const StringRef = extern struct {
```

Comment:
i'm not sure about this, the other way to handle StringRef is by returning the pointer from functions, and length as last argument.